pallets/drand/src/tests.rs

```diff
@@ -302,25 +337,25 @@ fn can_execute_and_handle_valid_http_responses() {
 		let mut state = state.write();
 		state.expect_request(PendingRequest {
 			method: "GET".into(),
-			uri: "https://api.drand.sh/52db9ba70e0cc0f6eaf7803dd07447a1f5477735fd3f661792ba94600c84e971/info".into(),
-			response: Some(QUICKNET_INFO_RESPONSE.as_bytes().to_vec()),
+			uri: "https://drand.cloudflare.com/52db9ba70e0cc0f6eaf7803dd07447a1f5477735fd3f661792ba94600c84e971/info".into(),
```

Comment:
We need to unhardcode this. Not necessarily in this PR, but maybe 
```suggestion
// TODO: use fetch_drand_chain_info in tests instead of hardcoded urls https://github.com/ideal-lab5/pallets/issues/28
			uri: "https://drand.cloudflare.com/52db9ba70e0cc0f6eaf7803dd07447a1f5477735fd3f661792ba94600c84e971/info".into(),
```